### PR TITLE
[Core][TrilinosApplication] Defining list of fastest direct linear solvers.

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Part of https://github.com/KratosMultiphysics/Kratos/pull/12309, defining list of fastest direct linear solvers.

**🆕 Changelog**

- [Defining list into space](https://github.com/KratosMultiphysics/Kratos/commit/e0a05c9901d6cdf5f4c60f0387b1c415223de22b)

### CODE REVIEW

**Comment by philbucher** [comment]
I think this is a good addition 

Can you please expose and use it in the python level to avoid code duplication?

**Comment by loumalouomega** [comment]
> I think this is a good addition
> 
> Can you please expose and use it in the python level to avoid code duplication?

Done

File: kratos/python/add_space_to_python.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// External includes
+
+// Project includes
+#include "includes/define_python.h"
+#include "python/add_space_to_python.h"
+#include "spaces/ublas_space.h"
+
+namespace Kratos::Python
+{
+void AddSpaceToPython(pybind11::module& m)
```
Comment (philbucher) [inline kratos/python/add_space_to_python.cpp]: I am 99% sure this exists already somewhere 

Probably in the addstrategiestopython

Having it separate makes sense but not duplicated

Comment (loumalouomega) [reply kratos/python/add_space_to_python.cpp]: A priori it is not, at least I couldn't find it. UblasSpace is old...

Comment (loumalouomega) [reply kratos/python/add_space_to_python.cpp]: Okay I found it, name is UblasSparseSpace

**Review by philbucher — APPROVED** [review]
Lgtm

**Pull request merged**